lib/qdrant-docker-image-ecr-kms-deployment-cdk-stack.ts

```diff
@@ -0,0 +1,34 @@
+import * as kms from 'aws-cdk-lib/aws-kms';
+import * as cdk from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import * as ecrDeploy from 'cdk-ecr-deployment';
+import * as ecr from 'aws-cdk-lib/aws-ecr';
+import { QdrantDockerImageEcrDeploymentCdkStackProps } from './QdrantDockerImageEcrDeploymentCdkStackProps';
+
+export class QdrantDockerImageEcrDeploymentCdkStack extends cdk.Stack {
+    constructor(scope: Construct, id: string, props: QdrantDockerImageEcrDeploymentCdkStackProps) {
```

Comment:
**Suggestion:** The class name `QdrantDockerImageEcrDeploymentCdkStack` is already used in another file. Rename this class to avoid a naming collision.
```suggestion
export class QdrantDockerImageEcrKmsDeploymentCdkStack extends cdk.Stack {
    constructor(scope: Construct, id: string, props: QdrantDockerImageEcrDeploymentCdkStackProps) {
        super(scope, id, props);
        ...
    }
}
```